tools/cmake/afr_metadata.cmake

```
@@ -358,6 +364,15 @@ function(afr_write_metadata)
     )
 endfunction()
 
+# This function is used for any additional files that need
+# be included in the ZIP downloaded from FreeRTOS console.
+function(afr_files_to_console_metadata)
+    set(prop_var AFR_FILES_TO_CONSOLE_METADATA)
+    set(${prop_var} "" CACHE INTERNAL "")
+    afr_cache_append(${prop_var} ${ARGN})
```

If this function is called multiple times in the same build combination, wouldn't the variable be overwritten?
It may be better to update this function to support appending to the `AFR_FILES_TO_CONSOLE_METADATA` list if called multiple times.
```suggestion
function(afr_files_to_console_metadata)
    set(prop_var AFR_FILES_TO_CONSOLE_METADATA)
    if( NOT DEFINED ${prop_var} )
       set(${prop_var} "" CACHE INTERNAL "")
    endif()
    afr_cache_append(${prop_var} ${ARGN})
```

---

You are right~